book/src/custom.md

```diff
@@ -0,0 +1,37 @@
+# Custom statements and custom operations
```

Comment:
How would you see doing the custom deductions section in a separate PR? I'm afraid that it might require a bit of iteration, and it may block the rest of the current PR, which ideally we could merge soon while we discuss and iterate the custom deductions PR.

Reply:
I agree!